test/e2e/storage/testsuites/multivolume.go

```diff
@@ -429,6 +429,10 @@ func (t *multiVolumeTestSuite) DefineTests(driver storageframework.TestDriver, p
 			e2eskipper.Skipf("Driver %q does not support multiple concurrent pods - skipping", dInfo.Name)
 		}
 
+		if l.driver.GetDriverInfo().Name == "vsphere" {
+			e2eskipper.Skipf("Driver %q does not support read only volumes - skipping", dInfo.Name)
+		}
+
 		// Create volume
```

Comment:
your change disable readonly tests for both fstype and block volumes, whereas what I was expecting is - that this test should be disabled only for raw-block *and* readonly volumes.

Reply:
My bad! I have tried to fix it properly now. PTAL...